js/context.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+module.exports = {
+  '@context': {
+    id: '@id',
+    type: '@type',
+    '@protected': true,
+
+    DeleteKeyOperation: 'https://w3id.org/security#DeleteKeyOperation',
+    DeriveSecretOperation: 'https://w3id.org/security#DeriveSecretOperation',
+    ExportKeyOperation: 'https://w3id.org/security#ExportKeyOperation',
+    GenerateKeyOperation: 'https://w3id.org/security#GenerateKeyOperation',
+    KmsOperation: 'https://w3id.org/security#KmsOperation',
+    RevokeKeyOperation: 'https://w3id.org/security#RevokeKeyOperation',
+    SignOperation: 'https://w3id.org/security#SignOperation',
+    UnwrapKeyOperation: 'https://w3id.org/security#UnwrapKeyOperation',
+    VerifyOperation: 'https://w3id.org/security#VerifyOperation',
+    WrapKeyOperation: 'https://w3id.org/security#WrapKeyOperation',
+    invocationTarget: {
+      '@id': 'https://w3id.org/security#invocationTarget', '@type': '@id'
+    },
+    kmsModule: 'https://w3id.org/security#kmsModule',
+    publicKey: 'https://w3id.org/security#publicKey',
+    signatureValue: 'https://w3id.org/security#signatureValue',
+    unwrappedKey: 'https://w3id.org/security#unwrappedKey',
+    verifyData: 'https://w3id.org/security#verifyData',
+    wrappedKey: 'https://w3id.org/security#wrappedKey'
```

Comment:
Just double checked... missing "revoked": https://github.com/digitalbazaar/webkms-switch/blob/main/lib/schemas/RevokeKeyOperation.js#L16

Did you check all the schemas?

Reply:
I looked at them just now and only saw `revoked` was missing ... but I'd like confirmation that at least one other person checked them.

Reply:
I think I checked all the schemas? I left `revoked` out because I didn't see it in security/v2. (But now I see that's because it was in security/v1.)

Reply:
@dlongley added `revoked`.